src/Features/CSharp/Portable/Completion/CompletionProviders/EnumAndCompletionListTagCompletionProvider.cs

```diff
@@ -237,7 +237,7 @@ private static async Task HandleSingleTypeAsync(CompletionContext context, Seman
                         rules: CompletionItemRules.Default,
                         contextPosition: position,
                         sortText: memberDisplayName,
-                        filterText: memberDisplayName));
+                        filterText: memberDisplayName).WithAdditionalFilterTexts(ImmutableArray.Create(staticSymbol.Name)));
```

Comment:
both changed here def deserve comments :)

Reply:
is there a test for this cse?

Reply:
I think I have added a couple of tests for C# in this PR

Reply:
i see the tests for enum members, but this is specifically about static fields/props for the "completion list tag" case.  :)

Reply:
this should cover it

https://github.com/dotnet/roslyn/pull/64261/files#diff-31303c421816ee9001acbb872ec31e6ef3d701cd04eba6f2a41d02e49d70ae21R3012

Reply:
perfect thanks! :)

Reply:
does the "completion list tag" case need to be updated/covered?  or does this work not apply to it?  thanks!

Reply:
I don't think it's necessary, given that I made no change to the logic of checking completionlist tag, and completionlist tag items are calculated the same way as general static fields/props (except some minor difference in trigger condition).

Reply:
hmm, it seems completion list tag is currently broken, I will fix and add tests in a separate PR